x-pack/test/security_solution_api_integration/test_suites/genai/invoke_ai/trial_license_complete_tier/configs/serverless.config.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../../../../config/serverless/config.base';
+
+export default createTestConfig({
+  kbnTestServerArgs: [
+    // used for connector simulators
+    `--xpack.actions.proxyUrl=http://localhost:6200`,
```

Comment:
Idk that I can use `get-port` here. I'm wondering if @YulNaumenko can speak to the importance of the dynamic port number here as I see your name in the git blame from the spot I stole this code from (`x-pack/test/alerting_api_integration/common/config.ts`) https://github.com/elastic/kibana/pull/75232/files#diff-1477d89e2965e56261180479a84bfb0e762793a73b88baa29300a7ed023d0f28R61-R62